channels-rke2.yaml

```diff
@@ -8,7 +8,7 @@ releases:
   - version: v1.20.9+rke2r2
     minChannelServerVersion: v2.6.0-alpha1
     maxChannelServerVersion: v2.6.99
-  - version: v1.21.3-rc7+rke2r2
+  - version: v1.21.4-rc1+rke2r1
```

Comment:
We want to add in another version of RKE2 to be able to test upgrades from 1 version to another. 

This shouldn't be replacing the existing 1.21 version.